README.md

```diff
@@ -23,4 +23,5 @@ Developer documentation is available in Javadoc format at [https://hms-networks.
 1. [sc-flexy-string-lib](https://github.com/hms-networks/sc-flexy-string-lib)
 2. [sc-flexy-datapoint-lib](https://github.com/hms-networks/sc-flexy-datapoint-lib)
 3. [sc-flexy-tag-info-lib](https://github.com/hms-networks/sc-flexy-tag-info-lib)
-4. [sc-flexy-file-util-lib](https://github.com/hms-networks/sc-flexy-file-util-lib)
\ No newline at end of file
+4. [sc-flexy-file-util-lib](https://github.com/hms-networks/sc-flexy-file-util-lib)
+5. [sc-flexy-logger-lib](https://github.com/hms-networks/sc-flexy-logger-lib)
```

Comment:
These changes also need to be reflected in the package.html file, as well as version number there incremented

Reply:
I was going to do this in a separate PR to not add commits to this

Reply:
I think @oliver-walker-hms  indicated he is making another pr to address that.